src/agent/src/storage/fs_handler.rs

```diff
@@ -50,6 +50,23 @@ impl StorageHandler for OverlayfsHandler {
                 .options
                 .push(format!("workdir={}", work.to_string_lossy()));
         }
+        storage
+            .driver_options
```

Comment:
Seems a `for loop` may get code easier for understanding:)

Reply:
Done.